docs/transpile/ai-transpiler-passes.mdx

```diff
@@ -101,3 +101,8 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
 These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+
+
+## Citation
+
+If you use any AI feature from the Qiskit transpiler service in your research, check out [the recommended citation](./qiskit-transpiler-service#citation).
```

Comment:
```suggestion
If you use any AI feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```

Reply:
> I'm not sure what the tone is supposed to be.

We should be using a consistent tone for all docs

Reply:
Updated to be more friendly